### ISSUE TITLE
API stability information (1.0.0)

### ISSUE THREAD

**Issue by wolverian** [apple/swift-distributed-tracing#61]
Do you have stability guarantees for this package, and/or guidance for library authors when they should add support for tracing in their stable releases?

**Comment by ktoso** [comment]
Hi there,
Thanks for the question :-)

As you'll notice by the version numbers the package is in a pre-1.0 phase, so there's no strong stability guarantees yet.

Having that said, with WWDC behind us, some of us are becoming more free again to work on this package again and focus on its stabilization. I can't commit to an exact timeline, but we expect to work on hitting the 1.0 relatively soon... 

I posted a similar update here on the forums: https://forums.swift.org/t/road-to-tracing-1-0/54160/6

Since then we have resumed active work on some remaining things, including otel attributes with @slashmo and as soon as we figure out the swift-log integration I think we'll be able to call a 1.0.

What's the use case where you're seeking to adopt and what guidance were you missing from the existing "[In-Depth Guide for: Library/Framework developers](https://github.com/apple/swift-distributed-tracing#library-framework-developers--instrumenting-your-software) part of the readme?
